tests/functional/tests.py

```diff
@@ -1672,7 +1674,7 @@ def main():
     try:
         access_key = os.getenv('ACCESS_KEY', 'Q3AM3UQ867SPQQA43P2F')
         secret_key = os.getenv('SECRET_KEY',
```

Comment:
BTW all the indentation changes I had to do since my `emacs` complained a lot, there are a lot of formatting errors in this file which I avoided but - these changes were automatically done by my editor.